### ISSUE TITLE
STAR WARS: The Old Republic (1286830)

### ISSUE THREAD

**Issue by okegima** [ValveSoftware/Proton#4326]
# Compatibility Report
- Name of the game with compatibility issues: Star Wars The Old Republic (SWTOR)
- Steam AppID of the game: 1286830

## System Information
- GPU: RX550
- Driver/LLVM version:
- Kernel version: 
- Link to full system information report as [Gist](https://gist.github.com/):

https://gist.github.com/okegima/6c5ff507e68d0701147d3c306afcea6c#file-gistfile1-txt

- Proton version:5.13-1

## I confirm:
- [*] that I haven't found an existing compatibility report for this game.
- [ ] that I have checked whether there are updates for my system available.

## Symptoms 

Can't use ctrl c for copy and ctrl v to past in game chat or mail. ctrl a not working too. Not able to copy a word and past inside game and from outside (text editor) to inside game. It work's well on outside game (runing ubuntu 20.04.1 LTS

No issue when launching game one win 7. 

## Reproduction

Yes, did change keyboard, but nothing happen.

**Comment by droidoz** [comment]
I have the same problem with cut and paste in swtor chat. Using Proton 5.0-9 or 5.13-1 on Steam .Didn't have the problem using Lutris.

**Comment by ptkato** [comment]
I'm encountering a weird behaviour on the game part, some things straight out don't appear, for example the characters' portraits:

![image](https://user-images.githubusercontent.com/10677806/109381059-e4f8b500-78cf-11eb-8102-c4a499e29caf.png)

What is weird is that sometimes rarely, they do appear. This ain't exclusive to the portraits only, other times, though less often, the character models themselves don't appear, my companion goes invisible, and even scenery vanishing in a few cutscenes, just to appear again in the next one. I didn't have that jankiness when I installed the standalone client through Lutris.

**Comment by kisak-valve** [comment]
Hello @ptkato, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by ptkato** [comment]
[Here you go.](https://gist.github.com/ptkato/5297fdb9566e04357a0e70da88e527db)

**Comment by TheDromundKaas** [comment]
For a few days now, I have the problem of the FPS of SWTOR often times dropping to single digits. It happens much more when moving. I've seen it on planets (wide open areas) as well as my personal starship (tiny closed off area). When using a full DXVK_HUD, I can see that the running equalizer goes into the red at that point. It almost looks like loading, but the assets of SWTOR are basically tiny and low resolution. I've also checked if it is shader compilation, but it doesn't seem to be, at least there's no message bottom left about compiling shaders when it happens.

As I am on a rolling release distribution, I could not specifically state if it started with any version of the proprietary NVidia driver, as it gets updated frequently. I was watching the issue to see if anybody else had the problem, but maybe the Linux population of that game is low.

A month ago I did not have that problem, everything in the game was silky smooth 200+ FPS.

[My system information](https://gist.github.com/TheDromundKaas/b899233b1c04bd2345c1dc4c8dd02fb0)

**Comment by SveSop** [comment]
Updated proton to proton-6.3-5 today, and i have severe stutters when turning around.
This is due to dxvk being updated to 1.9 with proton-6.3-5.

Downgrading dxvk (replacing the d3d dlls in `~/.steam/steam/steamapps/common/Proton 6.3/dist/lib64/wine/dxvk` and the 32-bit folder) with dxvk compiled from [https://github.com/ValveSoftware/dxvk/tree/f8a4ca555a6e5d89f5162a042bbae550902f4e49](https://github.com/ValveSoftware/dxvk/tree/f8a4ca555a6e5d89f5162a042bbae550902f4e49) - dxvk-1.8.1 fixes the problem.

I did bisect this around april, but have since forgotten the commit ID. (But have an approximate if interested)

**Comment by kisak-valve** [comment]
Hello @SveSop, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

It might be worthwhile to also mention the regression to upstream DXVK with what bisect info you have left.

**Comment by ptkato** [comment]
@SveSop, in the options, try disabling nameplate scaling. For some reason nameplate scaling destroys my FPS when moving the camera around.

**Comment by SveSop** [comment]
@ptkato 
> @SveSop, in the options, try disabling nameplate scaling. For some reason nameplate scaling destroys my FPS when moving the camera around.

Interesting! That did indeed help a lot with DXVK-1.9. I guess this "scaling function" is what bugs out then, cos i did not really have any issues with fps < DXVK-1.9.
Thanks :)

**Comment by ptkato** [comment]
I might be mistaken, but I think it has to do with the shader cache, every time something new appears, the thingy needs to compile the shaders, when the name renders, the game stutters, but then when you turn the camera, the name changes its size, then it needs to compile the shaders again, thus stuttering once more. I used to have a similar problem with in Path of Exile, when dynamic resolution was enabled.

**Comment by ptkato** [comment]
I might be mistaken, but I think it has to do with the shader cache, every time something new appears, the thingy needs to compile the shaders, when the name renders, the game stutters, but then when you turn the camera, the name changes its size, then it needs to compile the shaders again, thus stuttering once more. I used to have a similar problem in Path of Exile, when dynamic resolution was enabled.

**Comment by SveSop** [comment]
@ptkato No problem seeing that being the case... Its just that this is not such an issue with DXVK-1.8.1, so atleast something did change there when it comes to this dynamic thing.
Working with upstream DXVK atm, so will see if they can figure something out :)

**Comment by SveSop** [comment]
@kisak-valve This has now been closed upstream by: https://github.com/doitsujin/dxvk/pull/2126
And in the meantime, it is a viable fix to disable `Scale Nameplates with Distance` in the settings, as this seems to help a lot.

**Comment by ChaosBlades** [comment]
I have noticed that anything newer than 6.3-6 the game will crash to desktop after playing for several hours. This includes Proton Experimental and Proton 6.16-GE-1.
I can happily play for hours on end using 6.3-6.

**Comment by ChaosBlades** [comment]
I got an update to Mesa 21.2.1 two days ago and since then I have had no crashes on Proton 6.16-GE-1. It appears the issue was a driver issue for my 6900xt. I have not tried Proton Experimental.

**Comment by ChaosBlades** [comment]
I did test Proton Experimental shortly after my last post and it was working flawlessly but now it looks like things have broken again. I am now getting the following error message on Proton Experimental and Proton-6.18-GE-2 immediately after the recent updates. All other games I have tried are still working. 

![Screenshot from 2021-10-03 19-17-03](https://user-images.githubusercontent.com/7530545/135775322-d803c936-5564-4a4a-98f3-8a2da10f4f9e.png)

I continue to have a stable experience on Proton 6.3-7 playing SWTOR.

Edit: ...and as soon as I report the issue it is resolved. This was a XanMod Kernel bug. I just got an update to 5.14.9-xanmod2. It seems kernel 5.14 was released the same day I updated to 6.18 GE and a Proton Experimental update released. It is odd that this kernel bug affected SWTOR specifically on Experimental and GE specifically.

Here were the changes in this kernel update for thoroughness.
* 1d25d6095fa0 Linux 5.14.9-xanmod2
* f164cc43ea1a zstd: fix fall-through warnings
* 90383d79db7c futex2: Add sysfs entry for syscall numbers
* 442d05a80b7e kernel: Enable waitpid() for futex2
* e5a1578d65ce docs: locking: futex2: Add documentation
* fa85b67049cc futex2: Add compatibility entry point for x86_x32 ABI
* 5f5c48319897 futex2: Implement requeue operation
* 9970c3df4edb futex2: Implement vectorized wait
* 1ee81ce10706 futex2: Add support for shared futexes
* ea14247af977 futex2: Implement wait and wake functions
* 6afbd4f8cd3c futex: Implement mechanism to wait on any of several futexes
* 12aea22c5b44 futex*: remove FUTEX_WAIT_MULTIPLE operation and futex2 patchset

**Comment by kisak-valve** [comment]
STAR WARS: The Old Republic (1286830)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5392.
@vasiliyaltunin posted on 2021-12-08T21:06:41:

# Compatibility Report
- Name of the game with compatibility issues: Star Wars The Old Republic (SWTOR)
- Steam AppID of the game: 1286830

## System Information
- GPU: GTX 1050 Ti
- Driver/LLVM version: 470.57.02
- Kernel version: 5.10
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/vasiliyaltunin/c631b87ea1108ab89f62c04d1fe90e7b
- Proton version: Experemental, 6.3-8

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Colors depth seems to low, in Experemental and 6.3-8 in 5.13-6 all looks just fine

## Reproduction

here i make screenshot to show color problems:
![experemental](https://user-images.githubusercontent.com/417082/145284178-faa2193c-0c15-4691-8da9-394ccecc22b2.png)
![5136](https://user-images.githubusercontent.com/417082/145284190-77669e07-0968-4660-b0bb-9a594e54a4bf.png)
[steam-1286830.log](https://github.com/ValveSoftware/Proton/files/7678954/steam-1286830.log)

As you can see in experimental colors looks like in old pc games you can clearly see it in places in shadows in torso

**Comment by kisak-valve** [comment]
Hello @vasiliyaltunin, your log hints that you're using `PROTON_USE_WINED3D=1` in the game's launch options. Can you double check if this is the case and remove it? This runtime option is not supported and could be directly related to the issue you're seeing.

**Comment by vasiliyaltunin** [comment]
Yes! Thank you this is solved problem! On experementall all works just fine!

**Comment by juliovco** [comment]
[My system information](https://gist.github.com/juliovco/9d24955493e6a2f04725d1baef612f99.js)
Proton version: 6.3
My launcher has a warning at the top "An error has ocurred. Please exit and restart the SWTOR Laucher." If I try to login, it hangs forever.
launcher log has "ERROR Steam DLL failed to load"
It used to work before. Played with no issues around the beginning of December.
[launcher_20220106.log](https://github.com/ValveSoftware/Proton/files/7824983/launcher_20220106.log)
[steam-1286830.log](https://github.com/ValveSoftware/Proton/files/7824984/steam-1286830.log)

**Comment by kisak-valve** [comment]
Hello @juliovco, these look like the lines of interest from your log:
```
err:module:import_dll Library api-ms-win-crt-runtime-l1-1-0.dll (which is needed by L"C:\\windows\\system32\\VCRUNTIME140.dll") not found
err:module:import_dll Library api-ms-win-crt-heap-l1-1-0.dll (which is needed by L"C:\\windows\\system32\\VCRUNTIME140.dll") not found
err:module:import_dll Library api-ms-win-crt-string-l1-1-0.dll (which is needed by L"C:\\windows\\system32\\VCRUNTIME140.dll") not found
err:module:import_dll Library api-ms-win-crt-stdio-l1-1-0.dll (which is needed by L"C:\\windows\\system32\\VCRUNTIME140.dll") not found
err:module:import_dll Library api-ms-win-crt-convert-l1-1-0.dll (which is needed by L"C:\\windows\\system32\\VCRUNTIME140.dll") not found
err:module:import_dll Library VCRUNTIME140.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
err:module:import_dll Library api-ms-win-crt-stdio-l1-1-0.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
err:module:import_dll Library api-ms-win-crt-string-l1-1-0.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
err:module:import_dll Library api-ms-win-crt-heap-l1-1-0.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
err:module:import_dll Library api-ms-win-crt-convert-l1-1-0.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
err:module:import_dll Library api-ms-win-crt-runtime-l1-1-0.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
err:module:import_dll Library api-ms-win-crt-environment-l1-1-0.dll (which is needed by L"Z:\\home\\fnord\\.steam\\steamapps\\common\\Star Wars - The Old Republic\\steam.dll") not found
```

I read this as either Proton 6.3 on your system or the game's wineprefix is damaged or incomplete. I would first try to verify the integrity of Proton's files in Steam, and if that doesn't help, then move the game's wineprefix (`/home/fnord/.steam/steamapps/compatdata/1286830`) somewhere out of the way. It should be noted that removing the wineprefix might cause saves and settings to be lost, which is why I suggested moving instead of deleting the folder.

**Comment by ChaosBlades** [comment]
Launcher works for me on 6.3 and Experimental Bleeding Edge. Checked just now.

**Comment by juliovco** [comment]
Thank you both for the responses. I had already tried to verify integrity of both Swtor and Proton 6.3, but nothing changed. Also had tried uninstalling Swtor and installing again (I made a backup of the settings), nothing changed as well.
Now I tried playing another game (Aragami) which I also installed with Proton 6.3, and it played normally. But maybe this one doesn't use those ms-win-crt stuff, I don't know.
And now I tried changing Swtor to Proton 5.13 and it worked - so I'm happy, I can live with that. But then it seems there is something wrong with my Proton 6.3 installation. I tried uninstalling it through steam, but it failed with unexpected error. Is it because I have other games installed with this version? Do I have to change the proton version of those games to be able to uninstall it?

**Comment by kisak-valve** [comment]
As you expected, Proton is treated as a game dependency by Steam when games are configured to use it. (https://github.com/ValveSoftware/steam-for-linux/issues/7407)

Maybe give https://github.com/ValveSoftware/Proton/issues/4269#issuecomment-794164206 a try?

**Comment by juliovco** [comment]
I tried the solution from the other thread you linked (removing the dist folder of 6.3), then changed Swtor back to 6.3 to try again. Then the launcher worked, but when I logged in, the game became a black screen. I verified integrity of both Swtor and 6.3 files but nothing changed. So I changed the games that were using 6.3, and also my Steam Play default setting (it was set to 6.3 as well), and managed to uninstall 6.3. Started the game again to reinstall 6.3. This time the launcher didn't open anymore (tried multiple times). So I tried uninstalling both 6.3 and Swtor, deleted what remained of the 6.3 folder, and reinstalled Swtor (and 6.3 automatically). Same result, the launcher doesn't open. I click Play, nothing happens and after a long time the button turns green again.
[steam-1286830.log](https://github.com/ValveSoftware/Proton/files/7826286/steam-1286830.log)

**Comment by juliovco** [comment]
It's working. I uninstalled Proton 6.3, deleted the folder, installed it again clicking on the Install button, changed Swtor back to 6.3 and clicked play. I don't know what difference it would make, or if it's just random, but now it's working.

**Comment by ProjectSynchro** [comment]
## System Information
- GPU:  RTX 2070
- Driver/LLVM version:  510.47.03
- Kernel version:  5.16.9
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/ProjectSynchro/f0d4e4f102ca29c71a683ea49dd4df7e
- Proton version: 7.0-1

## Symptoms 
Publisher video only plays on first run of game in fresh prefix.

## Reproduction
1. Ensure there are no prior prefixes, and launch the game
2. Observe Bioware video
3. Relaunch game, observe no intro video

See below log for first run of game:
[steam-1286830-first-run.log](https://github.com/ValveSoftware/Proton/files/8103411/steam-1286830-first-run.log)

Log from second run of game in same prefix:
[steam-1286830-no-bioware-video.log](https://github.com/ValveSoftware/Proton/files/8103410/steam-1286830-no-bioware-video.log)

**Comment by ProjectSynchro** [comment]
## System Information
- GPU:  RTX 2070
- Driver/LLVM version:  510.47.03
- Kernel version:  5.16.9
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/ProjectSynchro/f0d4e4f102ca29c71a683ea49dd4df7e
- Proton version: 7.0-1

## Symptoms 
New cinematic 'Disorder' does not play in game. This occurs during gameplay when the mission to begin the expansion is triggered for the first time, and in the main menu

## Reproduction
1. Launch the game
2. Select 'Play Cinematic' on the main menu, select 'Disorder'
3. Observe no video playback

See below log:
[steam-1286830-play-new-cinematic.log](https://github.com/ValveSoftware/Proton/files/8103413/steam-1286830-play-new-cinematic.log)

**Comment by ProjectSynchro** [comment]
## System Information
- GPU:  RTX 2070
- Driver/LLVM version:  510.54
- Kernel version:  5.16.11
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/ProjectSynchro/f0d4e4f102ca29c71a683ea49dd4df7e
- Proton version: Proton Experimental (experimental-7.0-20220228)

## Symptoms 
Shadows appear to be corrupted in various places, this behaviour is not seen in Proton 7.0-1 so I see this as a regression.

## Reproduction
1. Launch the game
2. Start a new Sith Character, observe broken shadows outside the Sith Academy
Example:
![Screenshot_20220303_212600](https://user-images.githubusercontent.com/5131712/156687903-319212a6-6c21-46f0-85da-de836c7730f8.png)

Changing various graphics settings appears to fix certain areas, but breaks others.

What it looks like in Proton 7.0-1:
![image](https://user-images.githubusercontent.com/5131712/156688142-ec29386d-256f-48da-a59e-5e91dfbb61d1.png)


See below log:
[steam-1286830-experimental.log](https://github.com/ValveSoftware/Proton/files/8182590/steam-1286830-experimental.log)

This appears to be a DXVK issue (I'm assuming) should an issue be opened upstream?

**Comment by Culted** [comment]
i am getting some very rare crashes with Proton 7.0-1 and Experimental (also with Proton-GE), the log file is very large because these crashes can happen at various times and this time i captured it after about one or two hours so i uploaded the full log to my dropbox account https://www.dropbox.com/s/snlsbemykiqbpp8/steam-1286830.log?dl=0 and i attached a log file with only the first 100 lines and last 100 lines

[swtor.log](https://github.com/ValveSoftware/Proton/files/8506689/swtor.log)

**Comment by Blisto91** [comment]
@ProjectSynchro What are your graphical settings for reproducing the issue in your screenshot?
I will do some quick testing here later.

@Culted It's possible it could be memory related crashes since this appears to be a 32bit game? There's a PR open in the dxvk repo that could possibly help.
I will try to do a quick test if it appears to do anything for this game.

**Comment by ProjectSynchro** [comment]
> @ProjectSynchro What are your graphical settings for reproducing the issue in your screenshot? I will do some quick testing here later.

It's been a while since I last logged in and I have gotten new hardware since then.. However I was able to get that regardless of the settings I was using (at the time). I don't have NVIDIA hardware anymore so I can't get a 1:1 reproduction of the issue.

**EDIT:** I have relogged into the game and travelled to some areas that I used to see the issues with shadows, I cannot see any issues anymore (with Proton Experimental). This could be related to my changing to AMD hardware, or due to updates to dxvk.

**Comment by Blisto91** [comment]
@ProjectSynchro Roger. I have access to both a AMD and Nvidia setup. AMD is my main so if you know of anything specific with that just say so.
[A PR just got opened](https://github.com/doitsujin/dxvk/pull/2681) at the dxvk repo with a configuration option that fixes some stuff in this game, so I'm interested if that also helps here or if something else is required for this.

**Comment by elinmyr** [comment]
> **EDIT:** I have relogged into the game and travelled to some areas that I used to see the issues with shadows, I cannot see any issues anymore (with Proton Experimental). This could be related to my changing to AMD hardware, or due to updates to dxvk.

I had this issue awhile back, too. Windows 8.1, NVIDIA gpu. I was also using ReShade at the time. DXVK 1.10.1 resolved the issue for me. I just tested this a few minutes ago, because I've also done some software/driver updates since then.

DXVK 1.10
![dxvk before](https://user-images.githubusercontent.com/107258923/173347955-356c8454-5a41-45c3-a0b3-581374720b4b.png)

DXVK 1.10.1
![dxvk after](https://user-images.githubusercontent.com/107258923/173348013-3e5ff52b-59b9-4968-bb03-2060f977b0d4.png)

**Comment by Blisto91** [comment]
@ProjectSynchro @elinmyr 👍  Thanks for testing.

**Comment by kisak-valve** [comment]
Star Wars - The Old Republic Game Launcher stopped working

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6360.
@schwigi posted on 2022-12-03T09:28:58:

## Symptoms 
With the latest release of Proton Experimental (2022-12-01) the Game Launcher does not show up on screen. The process is running though and not crashing.

## Reproduce
Run the game with Proton Experimental (2022-12-01) on the Steam Deck. This happens on my Desktop PC as well.

## Used Hardware
1) Steam Deck, running Steam Beta channel
2) AMD Ryzen 3600, RX 580 running a Manjaro Linux

[steam-1286830.log](https://github.com/ValveSoftware/Proton/files/10146081/steam-1286830.log)


### Additional Note
The reasons for running Proton Experimental at all, are graphic error on the Steam Deck when using Proton 7.0-4 or Proton Next 7.0-5 during the cut scenes.

**Comment by alasky17** [comment]
@schwigi Thank you for reporting this!  We looked into this one and turns out it was an incorrect early version of a patch in experimental.  The upstream version in Wine should not have the same issue, and is already pushed to the bleeding-edge branch.  This will be fixed in the next official experimental update as well.

**Comment by schwigi** [comment]
Just checked on my end as well and the launcher is starting again with Proton Experimental on my Desktop. Also tested it on the Steam Deck (updated to Preview channel) and the launcher does not show yet.

Thanks for you work.

Edit: Tested again and it is working again on Steam Deck as well.

**Comment by Newbytee** [comment]
The launcher no longer starts up with Proton Experimental for me as of today. I was able to play it two days ago (18 jan 2023). Reverting to Proton 7.0-5 works around the issue and both the launcher and the game start up fine.

**Comment by Blisto91** [comment]
@Newbytee Hello and thanks for the report. I can reproduce the issue and will try to bisect it.

**Comment by Blisto91** [comment]
@Newbytee The Proton devs have fixed the regression and it should work now on bleeding edge. Will probably get pulled into next regular experimental update.

**Comment by ahjolinna** [comment]
swtor still wont launch for me (7.0-6, experimental-bleeding or GE-Proton7-49)

here is the log [steam-1286830.log](https://gist.githubusercontent.com/ahjolinna/feb48151c08d9fc8fa1d9024b528759b/raw/8a6c7e959eddbf559c0374a5c08315995404123d/steam-1286830.log)

----

my system spec:

```
             .;ldkO0000Okdl;.                
         .;d00xl:^''''''^:ok00d;.            OS: openSUSE MicroOS
       .d00l'                'o00d.          Kernel: x86_64 Linux 6.1.12-1-default
     .d0K^'  Okxoc;:,.          ^O0d.        Uptime: 3h 21m
    .OVVAK0kOKKKKKKKKKKOxo:,      lKO.       Packages: Unknown
   ,0VVAKKKKKKKKKKKKK0P^,,,^dx:    ;00,      Shell: bash 5.2.15
  .OVVAKKKKKKKKKKKKKk'.oOPPb.'0k.   cKO.     Resolution: 3072x1728
  :KVAKKKKKKKKKKKKKK: kKx..dd lKd   'OK:     DE: KDE 5.103.0 / Plasma 5.27.0
  lKlKKKKKKKKKOx0KKKd ^0KKKO' kKKc   lKl     WM: KWin_wayland
  lKlKKKKKKKKKK;.;oOKx,..^..;kKKK0.  lKl     GTK Theme: Breeze [GTK2],  [GTK3]
  :KAlKKKKKKKKK0o;...^cdxxOK0O/^^'  .0K:     Icon Theme: Papirus-Dark
   kKAVKKKKKKKKKKKK0x;,,......,;od  lKP      Disk: 16T / 21T (75%)
   '0KAVKKKKKKKKKKKKKKKKKK00KKOo^  c00'      CPU: AMD Ryzen 7 5700G with Radeon Graphics @ 16x 4.3GHz
    'kKAVOxddxkOO00000Okxoc;''   .dKV'       GPU: NVIDIA GeForce RTX 3060 Ti
      l0Ko.                    .c00l'        RAM: 5594MiB / 15779MiB
       'l0Kk:.              .;xK0l'         
          'lkK0xc;:,,,,:;odO0kl'            
              '^:ldxkkkkxdl:^'

**Comment by Blisto91** [comment]
@ahjolinna have you tried deleting the prefix and then verifying the game again afterwards?
Looking at your log paths i believe it should be located at `~/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/compatdata/1286830`

**Comment by ahjolinna** [comment]
> @ahjolinna have you tried deleting the prefix and then verifying the game again afterwards? Looking at your log paths i believe it should be located at `~/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/compatdata/1286830`

yeah I did, and I also tried it again ....did not help and I had the same error at the end of the log again

**Comment by ahjolinna** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4326#issuecomment-1439510055

okay I found the solution for my MicroOS issue or more exact SELinux issue

apparently I need to run this command: `sudo setsebool selinuxuser_execmod 1` to get the game run and other (mmo) games that had the same issue

more about it here: 
https://en.opensuse.org/Portal:MicroOS/Desktop#Steam_Proton,_Bottles,_WINE,_Lutris,_Android_Studio_emulator_not_working_from_flatpaks

**Comment by marcelomendes** [comment]
## Star Wars - The Old Republic Rendering issues with Proton 8
### Symptoms

With the latest release of Proton 8-x the game presents some rendering/lighting issues.

### Reproduce
Run the game with Proton 8-x (7-x works just fine)
Tested on a fresh prefix with the same results.

### Used Hardware

Operating System Version:
    Debian GNU/Linux 12 (bookworm) (64 bit)
    Kernel Name:  Linux
    Kernel Version:  6.1.0-8-amd64
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12101007
    X Window Manager:  GNOME Shell
    Steam Runtime Version:  steam-runtime_0.20230118.0

Video Card:
    Driver:  NVIDIA Corporation NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2
    Driver Version:  4.6.0 NVIDIA 525.105.17
    OpenGL Version: 4.6

### Workaround:
Fallback to Proton 7-x


Using Proton 8-x

![Screenshot from 2023-05-04 17-15-35](https://user-images.githubusercontent.com/178343/236332563-4ea7a9fd-7fab-4aeb-aecf-a539b596a37f.png)

Using Proton 7-x

![Screenshot from 2023-05-04 17-27-57](https://user-images.githubusercontent.com/178343/236334160-69d5188d-08c8-4877-866e-8e5687cdc94d.png)

### Log generated with PROTON_LOG

[steam-1286830.log.gz](https://github.com/ValveSoftware/Proton/files/11401842/steam-1286830.log.gz)

**Comment by kisak-valve** [comment]
Hello @marcelomendes, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

**Comment by marcelomendes** [comment]
@kisak-valve  Added the compressed log as requested.

**Comment by Blisto91** [comment]
@marcelomendes Hi there and thank you for the report. 

This is a Nvidia driver bug that got unveiled in a dxvk commit included in Proton 8. It has already been fixed in the beta drivers but have yet to arrive in the stable ones. A way to work around it besides playing with Proton 7 is to set the ingame AA to below high if i remember corectly. High and Very High include MSAA which triggers the issue.

**Comment by marcelomendes** [comment]
Setting AA to Medium solved it for now. Thank you @kisak-valve for the quick response!

**Comment by schwigi** [comment]
With the latest proton experimental build (2023-07-19) the camera jumps around crazy after the the right click. Camera movement without releasing the RMB is.

This happens on only my desktop (Manjaro Linux) running plasma-wayland. 

It is working fine on the steam deck and on my desktop when using X11. So there might be some issue how the newer wine handles wayland input.

**Comment by GregDuhamel** [comment]
Hello,

same here, i had to switch back to Proton 8.0-2 
I'm using Steam through [Flatpak](https://flathub.org/apps/com.valvesoftware.Steam)

Steam Runtime : https://gist.github.com/GregDuhamel/722edd6fb9faeebc1e628c5a34373d9d

<details><summary>System Information</summary>

```
```

</details>

**Comment by alasky17** [comment]
@schwigi @GregDuhamel Thank you so much for the reports!  You have indeed found a regression with one of the experimental patchsets that is currently a work in progress (hence the name "Proton Experimental" :D).  I have passed along the information so that the patchset can be refined and improved.  In the meantime, please continue to use Proton 8.0-2 for this title as this patchset will not be included in a stable release until we have squashed such regressions.

**Comment by GregDuhamel** [comment]
Thanks @alasky17 for your quick answer.

**Comment by schwigi** [comment]
Thanks as well @alasky17. Will use an older Version as well, until it is time to get experimental again.

**Comment by ProjectSynchro** [comment]
Hello, I am seeing a GPU hang when completing quests in both Proton 8.0-2 and Experimental (bleeding-edge).

It typically is reproducible when the 'quest reward'  page is supposed to open. Instead of opening, the GPU hangs, and X crashes. A reboot is necessary to recover from this. 

If you want to attempt to reproduce, try finishing any quest that has a reward.

I have attached a log from a run that crashed:
[steam-1286830.zip](https://github.com/ValveSoftware/Proton/files/12113974/steam-1286830.zip)

I am running an AMD RX 7900XTX with Mesa 23.2.0 (it's built from git, cannot recall which commit specifically)
System Information: https://gist.github.com/ProjectSynchro/f0d4e4f102ca29c71a683ea49dd4df7e

**Comment by Blisto91** [comment]
@ProjectSynchro Hello there.
I haven't been able to reproduce with the same GPU with either radv 23.2.0-rc1 or latest git. I tried completing a couple of quests on each with a rewards screen and also 1 quest before updating my git version.
Might be interesting if you could try a stable driver or newer git version.

**Comment by ProjectSynchro** [comment]
> @ProjectSynchro Hello there. I haven't been able to reproduce with the same GPU with either radv 23.2.0-rc1 or latest git. I tried completing a couple of quests on each with a rewards screen and also 1 quest before updating my git version. Might be interesting if you could try a stable driver or newer git version.

I was also beginning to suspect it's due to a faulty build of Mesa, especially once issues in other Vulkan applications started to manifest.

After a fresh rebuild from latest git, the issue seems to have disappeared..

If it starts happening again, I'll attempt to debug the issue and open a report upstream.

**Comment by alasky17** [comment]
@schwigi @GregDuhamel This regression should be fixed with the most recent experimental release (experimental-8.0-20230727).  Please let me know if there is any lingering regressive behavior :)

**Comment by GregDuhamel** [comment]
<details><summary>@alasky17 Got another issue on two different card (6600XT & 7900XT) it seems with DXVK : </summary>

```
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: [drm] Skip scheduling IBs!
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: recover vram bo from shadow done
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: recover vram bo from shadow start
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring mes_kiq_3.1.0 uses VM inv eng 14 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring jpeg_dec uses VM inv eng 4 on hub 1
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring vcn_unified_1 uses VM inv eng 1 on hub 1
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring vcn_unified_0 uses VM inv eng 0 on hub 1
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring sdma1 uses VM inv eng 13 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring sdma0 uses VM inv eng 12 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.3.1 uses VM inv eng 11 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.2.1 uses VM inv eng 10 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.1.1 uses VM inv eng 9 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.0.1 uses VM inv eng 8 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.3.0 uses VM inv eng 7 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.2.0 uses VM inv eng 6 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.1.0 uses VM inv eng 4 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring comp_1.0.0 uses VM inv eng 1 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: ring gfx_0.0.0 uses VM inv eng 0 on hub 0
22:59:03 kernel: amdgpu 0000:03:00.0: [drm:jpeg_v4_0_hw_init [amdgpu]] JPEG decode initialized successfully.
22:59:03 kernel: [drm] VCN decode and encode initialized successfully(under DPG Mode).
22:59:03 kernel: [drm] kiq ring mec 3 pipe 1 q 0
22:59:03 kernel: [drm] DMUB hardware initialized: version=0x07001A00
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: SMU is resumed successfully!
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: SMU driver if version not matched
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: smu driver if version = 0x00000037, smu fw if version = 0x0000003b, smu fw program = 0, smu fw version = 0x004e5500 (78.85.0)
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: SMU is resuming...
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: SECUREDISPLAY: securedisplay ta ucode is not available
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: RAP: optional rap ta ucode is not available
22:59:03 kernel: [drm] reserve 0x1300000 from 0x84fc000000 for PSP TMR
22:59:03 kernel: [drm] PSP is resuming...
22:59:03 kernel: [drm] VRAM is lost due to GPU reset!
22:59:03 kernel: [drm] PCIE GART of 512M enabled (table at 0x00000084FEB00000).
22:59:03 kernel: amdgpu 0000:03:00.0: amdgpu: GPU reset succeeded, trying to resume
22:59:02 kernel: amdgpu 0000:03:00.0: amdgpu: GPU smu mode1 reset
22:59:02 kernel: amdgpu 0000:03:00.0: amdgpu: GPU mode1 reset
22:59:02 kernel: amdgpu 0000:03:00.0: amdgpu: MODE1 reset
22:59:02 kernel: [drm:gfx_v11_0_hw_fini [amdgpu]] *ERROR* failed to halt cp gfx
22:59:02 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:02 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:02 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:02 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:02 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:02 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:02 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:02 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:02 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:02 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:01 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:01 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:01 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:01 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:01 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:01 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:01 kernel: [drm:amdgpu_mes_unmap_legacy_queue [amdgpu]] *ERROR* failed to unmap legacy queue
22:59:01 kernel: [drm:mes_v11_0_submit_pkt_and_poll_completion.constprop.0 [amdgpu]] *ERROR* MES failed to response msg=3
22:59:01 kernel: amdgpu 0000:03:00.0: amdgpu: soft reset failed, will fallback to full reset!
22:59:01 kernel: Failed to wait all pipes clean
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          RW: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MAPPING_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          PERMISSION_FAULTS: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          WALKER_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MORE_FAULTS: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          Faulty UTCL2 client ID: CB/DB (0x0)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00000000
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:40 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          RW: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MAPPING_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          PERMISSION_FAULTS: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          WALKER_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MORE_FAULTS: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          Faulty UTCL2 client ID: CB/DB (0x0)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00000000
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:40 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          RW: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MAPPING_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          PERMISSION_FAULTS: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          WALKER_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MORE_FAULTS: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          Faulty UTCL2 client ID: CB/DB (0x0)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00000000
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:40 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          RW: 0x1
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MAPPING_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          PERMISSION_FAULTS: 0x5
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          WALKER_ERROR: 0x0
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          MORE_FAULTS: 0x1
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:          Faulty UTCL2 client ID: CPC (0x5)
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00040A51
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:40 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
22:59:00 kernel: gmc_v11_0_process_interrupt: 55 callbacks suppressed
22:59:00 kernel: amdgpu 0000:03:00.0: amdgpu: IP block:gfx_v11_0 is hung!
22:58:59 kernel: amdgpu 0000:03:00.0: amdgpu: GPU reset begin!
22:58:59 kernel: [drm:amdgpu_job_timedout [amdgpu]] *ERROR* Process information: process swtor.exe pid 110324 thread dxvk-submit pid 110372
22:58:59 kernel: [drm:amdgpu_job_timedout [amdgpu]] *ERROR* ring gfx_0.0.0 timeout, signaled seq=98704404, emitted seq=98704406
``` 

</details>

Just to be sure any recent change on that side ?

**Comment by kisak-valve** [comment]
Hello @GregDuhamel, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the GPU reset, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by alasky17** [comment]
@GregDuhamel There are lots of changes in DXVK ... if you suspect a regression, it is fairly easy to confirm it is DXVK if you have a working Proton version to compare (eg 8.0-3).  
1) Save a copy of these folders: ~/.steam/steam/steamapps/common/Proton - Experimental/files/lib/wine/dxvk and ~/.steam/steam/steamapps/common/Proton - Experimental/files/lib64/wine/dxvk.  
2) cp ~/.steam/steam/steamapps/common/Proton 8.0/dist/lib/wine/dxvk/* ~/.steam/steam/steamapps/common/Proton - Experimental/files/wine/dxvk/.  and same for lib64 folder.
3) After you are done, re-replace the dxvk contents in Experimental with the original dxvk folders that you saved, and the run "Verify Integrity of Files" on Proton - Experimental to confirm that everything is back to default.

Also, it helps to also describe the behavior you are seeing in addition to posting a log :)

**Comment by GregDuhamel** [comment]
Sorry guys it's taking longer to provide you with trace, unfortunately or fortunately, i was not able to reproduce the crash during the last two or three days.

**Comment by GregDuhamel** [comment]
Here we go @kisak-valve , hopes this will be enough.

[Steam Runtime Diagnostics](https://gist.github.com/GregDuhamel/03478adf1377e6c7818708711eb307cd)
[steam-1286830.log.zip](https://github.com/ValveSoftware/Proton/files/12501742/steam-1286830.log.zip)

**Comment by GregDuhamel** [comment]
> @GregDuhamel There are lots of changes in DXVK ... if you suspect a regression, it is fairly easy to confirm it is DXVK if you have a working Proton version to compare (eg 8.0-3).
> 
>     1. Save a copy of these folders: ~/.steam/steam/steamapps/common/Proton - Experimental/files/lib/wine/dxvk and ~/.steam/steam/steamapps/common/Proton - Experimental/files/lib64/wine/dxvk.
> 
>     2. cp ~/.steam/steam/steamapps/common/Proton 8.0/dist/lib/wine/dxvk/* ~/.steam/steam/steamapps/common/Proton - Experimental/files/wine/dxvk/.  and same for lib64 folder.
> 
>     3. After you are done, re-replace the dxvk contents in Experimental with the original dxvk folders that you saved, and the run "Verify Integrity of Files" on Proton - Experimental to confirm that everything is back to default.
> 
> 
> Also, it helps to also describe the behavior you are seeing in addition to posting a log :)

Unfortunately 8.0-3 seems to have the same behavior for me.

It's hard to describe this issue,  it's so random, i can do like 7 PVP match and have no issue and then out of nowhere, always at the beginning of the match, game froze, host is unresponsive and i have to do an hard reboot.

**Comment by ProjectSynchro** [comment]
I'm beginning to see intermittent gpu hangs again in this title, annoyingly I cannot consistently reproduce the issue, it seems to happen during dialog.

In X, my entire PC needs to reboot as the session cannot recover from a driver reset, whereas in Wayland the session does recover at least.

Relevant dmesg logs:
```
[ 4997.557010] [drm:amdgpu_job_timedout [amdgpu]] *ERROR* ring gfx_0.0.0 timeout, signaled seq=77966164, emitted seq=77966166
[ 4997.557219] [drm:amdgpu_job_timedout [amdgpu]] *ERROR* Process information: process swtor.exe pid 7391 thread dxvk-submit pid 7432
[ 4997.557401] amdgpu 0000:0d:00.0: amdgpu: GPU reset begin!
[ 4998.570991] amdgpu 0000:0d:00.0: amdgpu: IP block:gfx_v11_0 is hung!
[ 4998.571158] gmc_v11_0_process_interrupt: 47 callbacks suppressed
[ 4998.571161] amdgpu 0000:0d:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:174 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
[ 4998.571165] amdgpu 0000:0d:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
[ 4998.571167] amdgpu 0000:0d:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00040B5D
[ 4998.571169] amdgpu 0000:0d:00.0: amdgpu:      Faulty UTCL2 client ID: CPC (0x5)
[ 4998.571170] amdgpu 0000:0d:00.0: amdgpu:      MORE_FAULTS: 0x1
[ 4998.571172] amdgpu 0000:0d:00.0: amdgpu:      WALKER_ERROR: 0x6
[ 4998.571173] amdgpu 0000:0d:00.0: amdgpu:      PERMISSION_FAULTS: 0x5
[ 4998.571174] amdgpu 0000:0d:00.0: amdgpu:      MAPPING_ERROR: 0x1
[ 4998.571175] amdgpu 0000:0d:00.0: amdgpu:      RW: 0x1
[ 4998.571180] amdgpu 0000:0d:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:174 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
[ 4998.571183] amdgpu 0000:0d:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
[ 4998.571184] amdgpu 0000:0d:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00000000
[ 4998.571186] amdgpu 0000:0d:00.0: amdgpu:      Faulty UTCL2 client ID: CB/DB (0x0)
[ 4998.571187] amdgpu 0000:0d:00.0: amdgpu:      MORE_FAULTS: 0x0
[ 4998.571188] amdgpu 0000:0d:00.0: amdgpu:      WALKER_ERROR: 0x0
[ 4998.571189] amdgpu 0000:0d:00.0: amdgpu:      PERMISSION_FAULTS: 0x0
[ 4998.571190] amdgpu 0000:0d:00.0: amdgpu:      MAPPING_ERROR: 0x0
[ 4998.571191] amdgpu 0000:0d:00.0: amdgpu:      RW: 0x0
[ 4998.571196] amdgpu 0000:0d:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:174 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
[ 4998.571198] amdgpu 0000:0d:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
[ 4998.571200] amdgpu 0000:0d:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00000000
[ 4998.571201] amdgpu 0000:0d:00.0: amdgpu:      Faulty UTCL2 client ID: CB/DB (0x0)
[ 4998.571202] amdgpu 0000:0d:00.0: amdgpu:      MORE_FAULTS: 0x0
[ 4998.571203] amdgpu 0000:0d:00.0: amdgpu:      WALKER_ERROR: 0x0
[ 4998.571204] amdgpu 0000:0d:00.0: amdgpu:      PERMISSION_FAULTS: 0x0
[ 4998.571205] amdgpu 0000:0d:00.0: amdgpu:      MAPPING_ERROR: 0x0
[ 4998.571206] amdgpu 0000:0d:00.0: amdgpu:      RW: 0x0
[ 4998.571211] amdgpu 0000:0d:00.0: amdgpu: [gfxhub] page fault (src_id:0 ring:174 vmid:0 pasid:0, for process  pid 0 thread  pid 0)
[ 4998.571213] amdgpu 0000:0d:00.0: amdgpu:   in page starting at address 0x0000000000000000 from client 10
[ 4998.571215] amdgpu 0000:0d:00.0: amdgpu: GCVM_L2_PROTECTION_FAULT_STATUS:0x00000000
[ 4998.571216] amdgpu 0000:0d:00.0: amdgpu:      Faulty UTCL2 client ID: CB/DB (0x0)
[ 4998.571217] amdgpu 0000:0d:00.0: amdgpu:      MORE_FAULTS: 0x0
[ 4998.571218] amdgpu 0000:0d:00.0: amdgpu:      WALKER_ERROR: 0x0
[ 4998.571219] amdgpu 0000:0d:00.0: amdgpu:      PERMISSION_FAULTS: 0x0
[ 4998.571220] amdgpu 0000:0d:00.0: amdgpu:      MAPPING_ERROR: 0x0
[ 4998.571221] amdgpu 0000:0d:00.0: amdgpu:      RW: 0x0
```
My current driver is built from mesa-git's HEAD

I am still running an AMD RX 7900XTX, (Kernel 6.5.1): https://gist.github.com/ProjectSynchro/f0d4e4f102ca29c71a683ea49dd4df7e

If I can catch this error again with `PROTON_LOG=1` I will update this post.

**Comment by GregDuhamel** [comment]
@ProjectSynchro did you get rid of it ? I'm still having some issue for nearly each dialog i'm looking at. Very difficult to play ...

**Comment by Blisto91** [comment]
Thank you for the reports. I have created a mesa issue to track the, at least rdna3, hang problem. https://gitlab.freedesktop.org/mesa/mesa/-/issues/9878

**Comment by GregDuhamel** [comment]
Thanks so much for this @Blisto91 ! Your workaround is working perfectly too !

**Comment by Blisto91** [comment]
Proper fix have now been merged in mesa https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25402

**Comment by ahjolinna** [comment]
does anyone else have flickering issue? especially if you play on "fullscreen (windowed)", in gameplay its not so bad but still noticeable/annoying compare to cinematic/conversation where it can be REALLY bad

[Steam Runtime Diagnostics](https://gist.githubusercontent.com/ahjolinna/fe1d05e309629fad0b7a49bcaa259173/raw/2681feef8253cc86b287bc81faa6e95640f5d63f/Steam%2520Runtime%2520Diagnostics)

[log file (with experimental) ](https://gist.githubusercontent.com/ahjolinna/15239b555d169039368d9f0615ae91cf/raw/daefc065aabf9141e0ccb33f193d9c4877c6a269/gistfile1.txt)

----

system spec:

```
Operating System: openSUSE MicroOS 20231226
KDE Plasma Version: 5.91.90
KDE Frameworks Version: 5.248.0
Qt Version: 6.6.1
Kernel Version: 6.6.7-1-default (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 5700G with Radeon Graphics
Memory: 46.8 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2
GPU Driver: v545.29.06

```

**Comment by Blisto91** [comment]
@ahjolinna Hi there. Are you able to provide a screenshot or video that shows the issue?

**Comment by ahjolinna** [comment]
@Blisto91 I wish I could do some video, but I use wayland (on nvidia) ...and proper screenshare/capture isn't there yet.

also its more of mix of stuttering & flickering

**Comment by ahjolinna** [comment]
@Blisto91 well I took a video with my phone. Here a new sith inq. characters first intro scene and you can see even from a phone recoding the issue...and this is even the mild version from what I have had :  

https://github.com/ValveSoftware/Proton/assets/6993072/922cbe0d-4d94-460d-8d23-f4e862cd9db2

**Comment by ahjolinna** [comment]
oh and for some reason everytime I login I have to re-accept the end-user & licenses agreement and ToS ...dont know why it wont remember it.

also according to their latest email they have a new 64bit client, maybe thats why there are some new issues (dont know when this happened, I just started to replay)

![image](https://github.com/ValveSoftware/Proton/assets/6993072/e81aa510-a33b-42ff-8967-b4cb10c3daac)

**Comment by Blisto91** [comment]
@ahjolinna Taking a look with my AMD GPU i am not reproducing.
Looking at your driver version and that you are using Nvidia with Wayland i believe it could be this issue https://forums.developer.nvidia.com/t/545-29-02-ghosting-artifacting-stuttering-on-fullscreen-when-below-monitor-framerate/271853


As for the EULA appearing each time, which i am seeing too, i believe this might just be the game as i am seeing reports of it in general when doing a quick search. Though i will try to poke at it a bit and see.

**Comment by elinmyr** [comment]
EULA thing happens to me on steam version any time I validate files. Presumably would happen on a new install as well.

**Comment by ProjectSynchro** [comment]
The EULA bug is reproducible on windows as well.. seems like they have forgotten to ship a file on Steam.. 

If you want to fix it, there's a workaround on this forum thread: https://forums.swtor.com/topic/931246-eula-bug-fix/

I did this myself to see if I could get it fixed and it seems to work, according to: https://www.reddit.com/r/swtor/comments/l101al/eula_agreement_every_time_i_open_the_game/k9lbdji/, it looks like the file is the same for everyone (the one my vanilla launcher generated is the same as the one that user posted on Reddit).

I attached the missing EULA.settings file if anyone needs it, it should workaround the EULA prompt when launching (rename when using it): [eula.settings.txt](https://github.com/ValveSoftware/Proton/files/14073634/eula.settings.txt)

**Comment by ProjectSynchro** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4326#issuecomment-1707623500

@Blisto91 I think the issue with DCC and RDNA3 has regressed with Mesa 25.2.7 and Proton `experimental-bleeding-edge-10.0-281237-20251210-pc6ead3-wf15b76-dd772a8-v54fd70`

```
[31194.321992] amdgpu 0000:0d:00.0: amdgpu: Dumping IP State
[31194.323221] amdgpu 0000:0d:00.0: amdgpu: Dumping IP State Completed
[31194.323231] amdgpu 0000:0d:00.0: amdgpu: [drm] AMDGPU device coredump file has been created
[31194.323233] amdgpu 0000:0d:00.0: amdgpu: [drm] Check your /sys/class/drm/card1/device/devcoredump/data
[31194.323235] amdgpu 0000:0d:00.0: amdgpu: ring gfx_0.0.0 timeout, signaled seq=42895807, emitted seq=42895809
[31194.323237] amdgpu 0000:0d:00.0: amdgpu:  Process swtor.exe pid 97150 thread dxvk-submit pid 97489
[31194.323239] amdgpu 0000:0d:00.0: amdgpu: Starting gfx_0.0.0 ring reset
[31196.323272] amdgpu 0000:0d:00.0: amdgpu: MES failed to respond to msg=RESET
[31196.323276] amdgpu 0000:0d:00.0: amdgpu: failed to reset legacy queue
[31196.323278] amdgpu 0000:0d:00.0: amdgpu: reset via MES failed and try pipe reset -110
[31196.323280] amdgpu 0000:0d:00.0: amdgpu: Ring gfx_0.0.0 reset failed
[31196.323283] amdgpu 0000:0d:00.0: amdgpu: GPU reset begin!
[31198.487140] amdgpu 0000:0d:00.0: amdgpu: MES failed to respond to msg=REMOVE_QUEUE
[31198.487145] amdgpu 0000:0d:00.0: amdgpu: failed to unmap legacy queue
[31198.733241] [drm:gfx_v11_0_hw_fini [amdgpu]] *ERROR* failed to halt cp gfx

```

I'm getting a pretty reproducible crash in the Sith Temple on Elom (during the Darth Nul questline from the latest release)
Is happening during dialog like before.

~Will try to see if the issue goes away with `RADV_DEBUG=nodcc`~ Edit: this game does not launch with either `nodcc`
or `nodisplaydcc` set, but does with *both* set. Even with them set, crashes occur even when not in dialog in the Temple at Elom.

**Comment by ProjectSynchro** [comment]
There appears to be other issues going on here related to Windowing. I nuked my wineprefix to test changes and the game is unable to go into fullscreen mode, and minimizing the black screen that spawns doesn't seem to work.

[steam-1286830.log](https://github.com/user-attachments/files/24084963/steam-1286830.log)

of note:
```
info:  Device reset
info:  D3D9DeviceEx::ResetSwapChain:
info:    Requested Presentation Parameters
info:      - Width:              5120
info:      - Height:             1440
info:      - Format:             D3D9Format::A8R8G8B8
info:      - Auto Depth Stencil: true
info:                  ^ Format: D3D9Format::D24S8
info:      - Windowed:           false
info:      - Swap effect:        1
info:  Setting display mode: 5120x1440@60
info:  Setting display mode: 5120x1440@60
35863.968:0124:0140:fixme:oleacc:find_class_data unhandled window class: L"#32769"
```

**Comment by FatherAlduin** [comment]
9070 XT, same issue as above, am able to relatively easily reproduce by initiating any of the class intro cutscenes. which matches with the pattern of it being triggered by dialogue.

```[  158.322726] amdgpu 0000:03:00.0: amdgpu: Dumping IP State
[  158.324182] amdgpu 0000:03:00.0: amdgpu: Dumping IP State Completed
[  158.324222] amdgpu 0000:03:00.0: amdgpu: [drm] AMDGPU device coredump file has been created
[  158.324223] amdgpu 0000:03:00.0: amdgpu: [drm] Check your /sys/class/drm/card1/device/devcoredump/data
[  158.324224] amdgpu 0000:03:00.0: amdgpu: ring gfx_0.0.0 timeout, signaled seq=111766, emitted seq=111768
[  158.324225] amdgpu 0000:03:00.0: amdgpu:  Process swtor.exe pid 7684 thread dxvk-submit pid 7733
[  158.324227] amdgpu 0000:03:00.0: amdgpu: Starting gfx_0.0.0 ring reset
[  160.530031] amdgpu 0000:03:00.0: amdgpu: Ring gfx_0.0.0 reset failed
[  160.530034] amdgpu 0000:03:00.0: amdgpu: GPU reset begin!
[  162.700593] amdgpu 0000:03:00.0: amdgpu: MES(1) failed to respond to msg=REMOVE_QUEUE
[  162.700597] amdgpu 0000:03:00.0: amdgpu: failed to unmap legacy queue
[  162.906144] [drm:gfx_v12_0_hw_fini [amdgpu]] *ERROR* failed to halt cp gfx
[  162.933781] amdgpu 0000:03:00.0: amdgpu: MODE1 reset
[  162.933782] amdgpu 0000:03:00.0: amdgpu: GPU mode1 reset
[  162.933822] amdgpu 0000:03:00.0: amdgpu: GPU smu mode1 reset
[  163.954936] amdgpu 0000:03:00.0: amdgpu: GPU reset succeeded, trying to resume
```
